xml/mysql_banners.xml

```diff
@@ -903,8 +904,132 @@
     <param pos="0" name="os.version" value="13.10"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.10"/>
   </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB.+~buster(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB on Debian 10 (Buster)</description>
+    <example service.version="10.3.20">5.5.5-10.3.20-MariaDB-1:10.3.20+maria~buster-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Debian"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="10.03"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:debian_linux:10.03"/>
```

Comment:
Here and elsewhere: How do we know it's a point release as opposed to just some version in a given release (Buster, Jessie, etc)?